webapp/src/client/client.js

```diff
@@ -20,7 +20,9 @@ export default class Client {
             topic,
             meeting_id: meetingId,
         });
-        return res.meeting_url;
+
+        const data = await res.json();
+        return data.meeting_url;
```

Comment:
I think the bug here is actually in `doPost`. We should be returning `response.json()` instead of just `response` in `doPost`

Reply:
@mickmister Fixed